docs/UserGuide.md

```diff
@@ -909,7 +928,7 @@ Features are built-in for the ease of use. They do not require any commands for
 
 ### Link to our User Guide: `help`
 
-Leads you to our user guide (i.e. this page) for guidance when using the application.
+Leads you to our user guide (i.e. this page) for guidance when using Hi:Re.
 
 Format: `help`
```

Comment:
According to the table of contents, `help` should be after export command, but now it seems to be under the `built-in features` section?